docker/README.md

```diff
@@ -28,41 +28,32 @@ docker run -it \
 
 ## Docker Compose
 
-To use the `docker-compose.yml` configuration in this repository, simply clone
-this repo and run
+The Docker Compose setup requires an Ethereum network name and node
+to connect to. By default, it will use `dev:http://docker.host.internal:8545`
+in order to connect to an Ethereum node running on your host machine.
```

Comment:
I would add "on port 8545" even though it's obvious, so people know they need to use that port (which is default for Ganache, but some ppl change it)